Riot/Modules/Analytics/Analytics.swift

```diff
@@ -200,6 +200,16 @@ extension Analytics {
         }
     }
     
+    /// Track when a user becomes unauthenticated without pressing the `sign out` button.
+    /// - Parameters:
+    ///   - reason: The error that occurred.
+    ///   - count: The number of times that error occurred.
```

Comment:
These might need updating :)

Reply:
Oh yea never came back to this. :) Both PRs actually need a self review before I open it up for review by others. Thanks for taking a look at the Analytics.